docker/jenkins-slave-npm/Dockerfile

```diff
@@ -1,6 +1,12 @@
-#invoke npm in jenkinsfile: sh "scl enable rh-nodejs6 'npm run build'" 
+#invoke npm in jenkinsfile: sh "scl enable rh-nodejs6 'npm run build'"
 FROM openshift/jenkins-slave-nodejs-rhel7:latest
 USER root
+
+ADD https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm google-chrome-stable_current_x86_64.rpm
+RUN yum -y install redhat-lsb libXScrnSaver xdg-utils wget
+RUN yum -y localinstall google-chrome-stable_current_x86_64.rpm
```

Comment:
nit pick: you can just `yum install -y https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm google-chrome-stable_current_x86_64.rpm`

Reply:
Thus this can be come a 1 liner yum install with the other packages